x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/styles.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { euiStyled } from '@kbn/react-kibana-context-styled';
+import { css } from '@emotion/react';
+import { EuiButtonEmpty, EuiContextMenu } from '@elastic/eui';
+import { euiThemeVars } from '@kbn/ui-theme';
+
+export const StyledContextMenu = euiStyled(EuiContextMenu)`
+  width: 250px;
+  & .euiContextMenuItem__text {
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  .euiContextMenuItem {
+    border-bottom: ${euiThemeVars.euiBorderThin};
+  }
+  .euiContextMenuItem:last-child {
+    border: none;
+  }
+`;
```

Comment:
`euiStyled` is marked as deprecated, we can apply these styles directly in the `EuiContextMenu` usage, since this `StyledContextMenu` is used in a single place

eg: 
```tsx
<EuiContextMenu
  css={css`
    width: 250px;
    & .euiContextMenuItem__text {
      overflow: hidden;
      text-overflow: ellipsis;
    }
    .euiContextMenuItem {
      border-bottom: ${euiThemeVars.euiBorderThin};
    }
    .euiContextMenuItem:last-child {
      border: none;
    }
  `}
   ...
```

Or create a ccs object similar to `groupCountCss` and use it in `EuiContextMenu` usage